### DISCUSSION TITLE
cn/2023/01/build-home-media-center-with-jellyfin-on-openwrt/

### DISCUSSION THREAD

**Discussion by giscus** [leovan/leovan.me#24] [Comments]
# cn/2023/01/build-home-media-center-with-jellyfin-on-openwrt/

历史尝试 入手 NAS 已经近 5 年的时间了，最初只是用来挂 PT 下载，然后在各种设备上通过 SMB 共享播放上面的视频。后面也尝试在利用 Plex 搭建家庭影音中心，但由于

https://leovan.me/cn/2023/01/build-home-media-center-with-jellyfin-on-openwrt/

**Comment by moonhangest** [comment]
师兄好优秀呀！我想打扰问一下，你建这个个人网站，一年要多少钱？一定要买服务器吗？😂

**Reply by moonhangest** [reply]
我留意你有关强化学习的内容很久了，一直在关注😀

**Reply by leovan** [reply]
除了域名以外，其他都是免费的。网站[源代码](https://github.com/leovan/leovan.me)直接放在了 Github 上，用的是 Hugo，当然还有很多其他框架。自动集成部署用的是 Cloudflare 的 [Pages](https://pages.cloudflare.com/)，free plan 流量完全够用，只是国内访问会稍微慢一些，也是支持很多静态网站框架部署的。如果不需要自己的域名，例如 Cloudflare Pages 提供 xxx.pages.dev 二级域名，那么整套网站就全部免费了。

**Comment by yydschen** [comment]
大佬主题修改的很棒~收藏了。😁

**Comment by charlie-ssong** [comment]
大佬，能问下jellyfin中nfo文件怎样设置保存路径

**Reply by leovan** [reply]
「保存路径」是指 Jellyfin 自己刮削的结果 nfo 存在哪里吗？这个还不是很清楚，我理解应该也是视频文件的相同路径下吧。我是使用 TMM 进行刮削的，Jellyfin 默认会读取视频文件路径下 `<filename>.nfo` 或是 `movie.nfo` 之类的文件。

**Comment by flyawayzack** [comment]
必须手动给你点个赞，找遍了整个全网的中文信息，只有你这里有这篇手把手的教程，非常详细而流程思路清晰。首先，我们的考虑思路是完全一致的：“由于 NAS 的 CPU 性能并不高，为了不给 NAS 其他功能带来过多压力，同时考虑软路由性能过剩，因此决定将 Jellyfin 安装在软路由上，再将 NAS 的资源挂载到软路由来实现整体解决方案。”但是我刚开始是试图在已运行的jellyfin上挂载smb协议的nas盘来看看能不能挂到Jellyfin的媒体库上，但是始终失败。后来看了你的文章尝试先打开了NFS的协议后，然后挂载到正在运行的jellyfin上，仍然失败。索性全部按照你的思路重来，关掉容器，删除镜像，先挂载好NFS协议的盘，然后再拉取和运行jellyfin的镜像。终于成功了。非常感谢！

我和你的经历正好是倒过来，我是学摄影，导演、艺术出身，平时喜欢研究折腾网络应用。2017年8月我也是在墨尔本开学术会议，也算是时空中交错的有缘人。感谢你详尽而细致的教程！

**Reply by leovan** [reply]
很高兴能够帮助到你，没事儿喜喜欢拍拍照片剪剪视频，有机会认识下专业大佬 🤝

**Comment by ErDong-Sir** [comment]
大佬，N5101可以硬解4K吗

**Reply by leovan** [reply]
对 CPU 不是很了解，可以去 Intel 官网看看是否支持，或者搭建一下试试看。
